examples/src/main/scala/org/apache/ignite/scalar/examples/spark/ScalarSharedRDDExample.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.scalar.examples.spark
+
+import org.apache.ignite.configuration.IgniteConfiguration
+import org.apache.ignite.spark.{IgniteContext, IgniteRDD}
+import org.apache.spark.{SparkConf, SparkContext}
+
+
+/**
+  * This example demonstrates the simplest code that creates the IgnitedRDD and
+  * shares it with multiple spark workers. The goal of this particular
+  * example is to provide the simplest code example of this logic.
+  * <p/>
+  * Remote nodes should always be started with special configuration file which
```

Comment:
I don't think that we need to mention remote nodes here since the example starts embedded Ignite nodes for every spark worker. So, I would remove the remote nodes related paragraphs from the doc and elaborate more on Ignite "standalone" flag that is set to "false" for the purpose of this example.